libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -54,9 +56,51 @@ export class ReadingListEffects implements OnInitEffects {
     )
   );
 
+  showSnackBarOnAdd$=createEffect(() => 
+    this.actions$.pipe(
+      ofType(ReadingListActions.confirmedAddToReadingList),
+      concatMap((action) => {
+        const typeItem:ReadingListItem = {
```

Comment:
Instead of typeItem, you can give name like book or readingListItem

Reply:
Changed typeItem to readingListItem